test/e2e/page-objects/pages/header-navbar.ts

```diff
@@ -69,13 +69,19 @@ class HeaderNavbar {
     await this.driver.waitForSelector('.multichain-account-menu-popover__list');
   }
 
-  async openAccountDetailsModal(): Promise<void> {
+  async openAccountDetailsModalDetailsTab(): Promise<void> {
     console.log('Open account details modal');
     await this.openThreeDotMenu();
     await this.driver.clickElement(this.openAccountDetailsButton);
```

Comment:
nit: maybe we can do this instead:


```suggestion
    await this.openAccountDetailsModal()
```